ping.cpp

```diff
@@ -345,6 +348,7 @@ bool ping_start(IPAddress adr, int count=0, int interval=0, int size=0, int time
     return false;
 }
 
+/*
 bool ping_regist_recv(struct ping_option *ping_opt, ping_recv_function ping_recv)
 {
     if (ping_opt == NULL)
```

Comment:
this need to be used if ye want other stats. 
Bad idea to remove it.

Reply:
So it means that  ping.cpp - ping_start() should be used by :
 - ESP32Ping.cpp - PingClass::ping() including a callback trigger
 - ping.cpp - ping()  , as standaloun ping, without the callback trigger
Do I understand correctly ?

Reply:
PS. 
Currently ping_regist_recv() & ping_regist_sent() are not used. 
Registering the receive callback is done by:
```C
// Callbacks
    _options.recv_function = reinterpret_cast<ping_recv_function>(&PingClass::_ping_recv_cb);
    _options.sent_function = NULL; //reinterpret_cast<ping_sent_function>(&_ping_sent_cb);
```
But I don't have a problem with them, I just wanted to avoid confusing other developers in future.